### PR TITLE
[Draft] Update todo.py

### PR DESCRIPTION
**Pull request by PLChris**
According to this forum discussion: https://forum.freecadweb.org/viewtopic.php?p=365865#p365865 the debug message should be commented out.

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [x] Branch rebased on latest master `git pull --rebase upstream master`
- [x] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

File: src/Mod/Draft/draftutils/todo.py [hunk 1]
```diff
@@ -110,12 +110,12 @@ def doTasks():
...
-              "commitlist: {1}\n"
-              "afteritinerary: {2}\n".format(todo.itinerary,
-                                             todo.commitlist,
-                                             todo.afteritinerary))
+#        print("Debug: doing delayed tasks.\n"
+#              "itinerary: {0}\n"
+#              "commitlist: {1}\n"
+#              "afteritinerary: {2}\n".format(todo.itinerary,
+#                                             todo.commitlist,
+#                                             todo.afteritinerary))
```
Comment (vocx-fc) [inline src/Mod/Draft/draftutils/todo.py]: Maybe you can do it like this:
```py
        DEBUG = 0
        if DEBUG:
            print("Debug: doing delayed tasks.\n"
                  "itinerary: {0}\n"
                  "commitlist: {1}\n"
                  "afteritinerary: {2}\n".format(todo.itinerary,
                                                 todo.commitlist,
                                                 todo.afteritinerary))
```
Then somebody can change the value to `DEBUG = 1` to reactivate the debug information quickly.

**Comment by vocx-fc** [comment]
I opened a new pull request #3001 to implement my proposed solution.

File: src/Mod/Draft/draftutils/todo.py (hunk 1 above)
Comment (yorikvanhavre) [reply src/Mod/Draft/draftutils/todo.py]: good idea...

**Comment by yorikvanhavre** [comment]
Since there is a refined version merged with #3001 I'll close this one...

**Pull request closed without merging**